sycl/test-e2e/BFloat16/bfloat16_vec.cpp

```diff
@@ -6,9 +6,8 @@
 //
 //===----------------------------------------------------------------------===//
 
-// TODO currently the feature isn't supported on FPGA.
 // TODO enable opaque pointers support on CPU.
-// UNSUPPORTED: cpu || accelerator
+// UNSUPPORTED: cpu
```

Comment:
It would need to be done in a different PR because this one is about accelerator, but is this still unsupported on CPU?

Reply:
The CPU is allegedly disabled for this due to opaque pointers support problems, which is tracked elsewhere. I will confirm, but this PR primarily focused on accelerator.